tests/cypress/integration/aliases_spec.js

```diff
@@ -0,0 +1,141 @@
+describe('Test aliases to H.O.C. (default)', () => {
+  beforeEach(() => {
+    cy.clearCookie('okta_help_user_lang')
+  })
+
+  it('when accessing H.O.C. home', () => {
+    cy.visit('okta_help.htm?id=csh-index')
+    cy.url().should('include', 'en-us/Content/index.htm')
+  })
+
+  it('when accessing ASA home', () => {
+    cy.visit('okta_help.htm?type=asa&id=csh-asa-overview')
+    cy.url().should('include', 'asa/en-us/Content/Topics/Adv_Server_Access/docs/asa-overview.htm')
+  })
+
+  it('when accessing Classic home', () => {
+    cy.visit('okta_help.htm?id=ext-admin-landing')
+    cy.url().should('include', 'en-us/Content/index-admin.htm')
+  })
+
+  it('when accessing End User home', () => {
+    cy.visit('okta_help.htm?type=eu&id=csh-user-home')
+    cy.url().should('include', 'eu/en-us/Content/Topics/end-user/end-user-home.htm')
+  })
+
+  it('when accessing OAG home', () => {
+    cy.visit('okta_help.htm?type=oag&id=ag-main')
+    cy.url().should('include', 'oag/en-us/Content/Topics/Access-Gateway/ag-main.htm')
+  })
+
+  it('when accessing OIE home', () => {
+    cy.visit('okta_help.htm?type=oie&id=csh-oie')
+    cy.url().should('include', 'oie/en-us/Content/Topics/identity-engine/oie-index.htm')
+  })
+
+  it('when accessing Workflows home', () => {
+    cy.visit('okta_help.htm?type=wf&id=ext-Okta-workflows')
+    cy.url().should('include', 'wf/en-us/Content/Topics/Workflows/workflows-main.htm')
+  })
+
+  it('when accessing "Directory_People"', () => {
+    cy.visit('okta_help.htm?type=oie&id=Directory_People')
+    cy.url().should('include', 'oie/en-us/Content/Topics/users-groups-profiles/usgp-people.htm?cshid=Directory_People#Directory_People1')
```

Comment:
interesting this page still has a `cshid` in the url... do you know why?

Reply:
Ah, I pulled that directly from the existing PDV test sometime ago. I can revise (here and the other two).

Reply:
That is to say, all of the returned URLs in have the `?cshid` query param, but because we're using the `should('include'` argument, the URL match can be partial ("the URL should include the following string"), thus omitting the query param portion is allowed.
I think this is OK here because we are checking that a user is directed to the expected page and not 404ing.